bin/handler-scale-down-kube.rb

```diff
@@ -0,0 +1,121 @@
+#! /usr/bin/env ruby
+#
+#  handler-scale-down-kube
+#
+# DESCRIPTION:
+# => Decreases the desired capacity of an AutoscalingGroup
+#
+# OUTPUT:
+#   plain text
+#
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#   gem: sensu-plugin
+#   gem: aws-sdk
+#
+# USAGE:
+# -j JSONCONFIG - The name of a json config file to be used
+#
+# NOTES:
+#
+# LICENSE:
+#   Brian Sizemore <bpsizemore@gmail.com>
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+#
+
+require 'aws-sdk'
+require 'json'
+require 'sensu-handler'
+
+class KubeScaler < Sensu::Handler
+  option :json_config,
+         description: 'Name of the json config file',
+         short: '-j JSONCONFIG',
+         long: '--json JSONCONFIG',
+         default: 'kube_scaler'
+
+  def autoscaling_group
+    get_setting('autoscaling_group')
+  end
+
+  def json_config
+    cli = KubeScaler.new
+    cli.config[:json_config]
+  end
+
+  def get_setting(name)
+    config_file = File.read("#{json_config}.json")
+    config = JSON.parse(config_file)
+    config[name]
+  end
+
+  def handle
+    $ASG = autoscaling_group
+    $AUTOSCALING = Aws::AutoScaling::Client.new
+    if !out_of_cooldown
+      puts 'An autoscaling event took place within the past 15 minutes. No action will be taken.'
+    else
+      puts 'No event has taken place within the past 15 minutes. Proceeding...'
+      begin_scaling
+    end
+  end
+
+  def asg_min_instances
+    resp = $AUTOSCALING.describe_auto_scaling_groups(auto_scaling_group_names: [$ASG],
+                                                     max_records: 1)
+    resp = resp[0][0][3]
```

Comment:
no need to assign this and than return it
```
resp = resp[0][0][3]
resp
```

should become
```
resp[0][0][3]
```

Is it possible for resp to be nil? or resp[0] to be nil?